.gitignore

```diff
@@ -119,8 +119,6 @@ dist
 # Generated by Cargo
 # will have compiled files and executables
 # /target/
-# Compiled wasm targets
-grpc/cargo-casper/*
```

Comment:
I had introduced this accidentally because I compiled the client contracts and my git was marking them as untracked. Sorry about that

Reply:
Yeah - not a problem.  I renamed the cargo-casper folder recently which is part of the issue :)